packages/css-library/dist/stylesheets/utilities.css

```diff
@@ -14341,10 +14341,10 @@ through all possible variants
   .medium-screen\:vads-u-order--initial {
     order: 0 !important;
   }
-  .medium-screen\:vads-u-font--sans {
+  .medium-screen\:vads-u-font-family--sans {
```

Comment:
These are generating for the `medium-screen` breakpoint because our workaround for the medium-screen utilities does not take into account the `responsive` setting for those utilties we don't want to generate for breakpoints.

https://github.com/department-of-veterans-affairs/component-library/blob/b057e59ee4357a0b925bb19d8e4897bf35c5388f/packages/css-library/src/utilities/settings.scss#L55-L62

When the custom breakpoints feature is available from USWDS, this will be removed.